### PR TITLE
add vs16.10 and vs.16.9 channels

### PR DESCRIPTION
**Pull request by epananth**
## Description

Msbuild needs these channels, I made temporary fix in msbuild post-build.yml but this will unblock them for good and they can start getting arcade updates. 

## Customer Impact

MSBuild builds will break,

## Regression

Made the same changes in MSbuild (post-build.yml) to unblock them, that is working

## Risk

How risky is this change?
Nope

## Workarounds

Are there available workarounds for the bug?

For now I made the change in the repo itself, and stopped the arcade update.

### CODE REVIEW

**Comment by epananth** [comment]
cc: @BenVillalobos

File: eng/common/templates/post-build/post-build.yml [hunk 1]
```diff
@@ -94,7 +96,7 @@ stages:
...
+                -AvailableChannelIds ${{parameters.NetEngLatestChannelId}},${{parameters.NetEngValidationChannelId}},${{parameters.NetDev5ChannelId}},${{parameters.NetDev6ChannelId}},${{parameters.GeneralTestingChannelId}},${{parameters.NETCoreToolingDevChannelId}},${{parameters.NETCoreToolingReleaseChannelId}},${{parameters.NETInternalToolingChannelId}},${{parameters.NETCoreExperimentalChannelId}},${{parameters.NetEngServicesIntChannelId}},${{parameters.NetEngServicesProdChannelId}},${{parameters.NetCoreSDK313xxChannelId}},${{parameters.NetCoreSDK313xxInternalChannelId}},${{parameters.NetCoreSDK314xxChannelId}},${{parameters.NetCoreSDK314xxInternalChannelId}},${{parameters.VS166ChannelId}},${{parameters.VS167ChannelId}},${{parameters.VS168ChannelId}},${{parameters.VSMasterChannelId}},${{parameters.VS169ChannelId}},${{parameters.VS1610ChannelId}}
```
Comment (riarenas) [inline eng/common/templates/post-build/post-build.yml:99]: Is the removal of some channels intentional here?

**Review by riarenas — APPROVED** [review]
@epananth can you fill out the template https://github.com/dotnet/arcade/blob/master/Documentation/Policy/AskModeTellModeTemplate.md since this is a release branch? You'll also need approval from Matt or Mark before merging.

File: eng/common/templates/post-build/post-build.yml (hunk 1 above)
Comment (epananth) [reply eng/common/templates/post-build/post-build.yml]: I reverted that. that was a bad merge

**Comment by markwilkie** [comment]
I'll let @mmitche approve.  (I don't see any issues)

**Review by mmitche — APPROVED** [review]

**Pull request merged**